test/js/source/tile_pyramid.test.js

```diff
@@ -313,8 +313,8 @@ test('TilePyramid#update', function(t) {
 
     t.test('retains parent tiles for pending children (wrapped)', function(t) {
         var transform = new Transform();
-        transform.width = 512;
-        transform.height = 512;
+        transform.width = 511;
+        transform.height = 511;
```

Comment:
See commit message here. https://github.com/mapbox/mapbox-gl-js/commit/e5ee155af3154aacbbb8f54dd700ff5767b9719d The tests were too sensitive.